apps/rush-lib/src/logic/PublishUtilities.ts

```diff
@@ -293,9 +295,10 @@ export class PublishUtilities {
     );
   }
 
-  private static _updateCommitDetails(filename: string, changes: IChangeInfo[] | undefined): void {
+  private static _updateCommitDetails(git: Git, filename: string, changes: IChangeInfo[] | undefined): void {
     try {
-      const fileLog: string = execSync('git log -n 1 ' + filename, {
+      const gitPath: string = git.getGitPathOrThrow();
+      const fileLog: string = execSync(`${gitPath} log -n 1 ${filename}`, {
```

Comment:
Should this use `Executable.spawnSync` to match elsewhere?

Reply:
Probably, but I don't want to change this code too much when publishing is going to be redesigned eventually anyway.